pipelines/timeseries_forecasts.R

```diff
@@ -142,9 +146,19 @@ cdc_flat_forecast <- function(data,
   return(cdc_flat_forecast)
 }
 
-main <- function(model_run_dir, n_forecast_days = 28, n_samples = 2000) {
+main <- function(
+    model_run_dir, n_forecast_days = 28, n_samples = 2000,
+    epiweekly = FALSE) {
+  prefix <- ""
+  dataname <- "data"
+  data_frequency <- "1 day"
+  if (epiweekly) {
+    dataname <- glue::glue("epiweekly_{dataname}")
+    prefix <- "epiweekly_"
+    data_frequency <- "1 week"
+  }
```

Comment:
I think either
```r
if (epiweekly) {
  # Assign variables
} else {
  # Assign variables
}
```
or separate
```r
prefix <- if_else(epiweekly, "epiweekly_", "")
```
is easier to read

Reply:
Agreed. Much nicer. I've changed this now.